plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.properties

```diff
@@ -3,14 +3,10 @@ cloudsdkdeploy=App Engine Project Deploy
 pluginName=Google App Engine Standard WTP Tools
 providerName=Google, Inc.
 
-runtimeTypeDescription=Google App Engine Standard Runtime (Custom)
-runtimeTypeName=App Engine Standard (Custom)
+runtimeTypeDescription=Google App Engine Standard Runtime
 runtimeTypeVendor=Google
 
-apptoolsTypeDescription=Google App Engine Standard Runtime
 apptoolsTypeName=App Engine Standard
```

Comment:
I'm wondering if the `apptools` prefix is descriptive enough... Does not seem to provide any useful.